app/templates/components/add-env-var.hbs

```diff
@@ -23,7 +23,29 @@
       showValidationStatusIcons=false
       as |field|
     }}
-      {{field.input  class='env-value' placeholder='Value'}}
+      {{field.input class='env-value' placeholder='Value'}}
+    {{/form.field}}
+  </div>
+   <div class="form-elem">
+    {{#form.field
+      label='Branch'
+      value=branch
+      required=false
+      class='env-branch-selector'
+      showValidationStatusIcons=false
+      as |field|
+    }}
+      {{#field.select
+        searchField='name'
+        placeholder='all branches'
+        allowClear=true
+        selected=branch
```

Comment:
This doesn't need to be specified directly, `form-field` component binds it to `value` internally.
```suggestion
```